modules/sdk/project-templates/project-templates-social-bookmark/build.gradle

```diff
@@ -1,4 +1,15 @@
 dependencies {
-	compileOnly group: "com.liferay", name: "com.liferay.project.templates", version: "4.1.6"
+	compile project(":sdk:project-templates:project-templates")
```

Comment:
If it doesn't break any functionality, please apply this change to all the other `project-template` subprojects for consistency.